daisy_workflows/image_import/enterprise_linux/translate.py

```diff
@@ -193,7 +193,7 @@ def yum_install(g, *packages):
 
   Args:
     g (guestfs.GuestFS): A mounted GuestFS instance.
-    *packages (list of str): The YUM packages to be installed.
+    *packages: The YUM packages to be installed.
```

Comment:
what is the type of this parameter now?

Reply:
Reverted back.

More info: It's a vararg, where each member is expected to be a string. I originally removed this type signature since my static analyzer said it was invalid,  but I couldn't find any PEPs or examples showing the correct approach. Anyway, regardless of the static analyzer, I think it's clearer for the human.